sdk/requirements.txt

```diff
@@ -7,3 +7,4 @@ pandas==1.2.2
 python-dateutil
 requests==2.25.1
 urllib3 >= 1.25.3
+cloudpickle==1.2.2
```

Comment:
This has now been pinned here as a fix for the pyfunc ensembler engine compatibility issues.